metric_providers/powermetrics/provider.py

```diff
@@ -181,7 +217,14 @@ def read_metrics(self, run_id, containers=None):
     def get_stderr(self):
         stderr = super().get_stderr()
 
-        if stderr is not None and str(stderr).find('proc_pid') != -1:
+        if stderr is not None and str(stderr).find('proc_pid') != -1 :
```

Comment:
Will that really do what we want? How I understand it you are returning here if the string "proc_pid" is in stderr.

However actually we want to return if ONLY this string is in stderr, not?

So we would have to iterate over all the lines and match?